ControlWork/Queue.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace ControlWork;
+
+public class ThreadSafetyQueue<T, P>
```

Comment:
```suggestion
public class ThreadSafetyQueue<Value, Priority>
```
Имена типовых переменных - это имена. Поэтому они должны быть содержательными.

Reply:
-0.5